src/test/java/de/dentrassi/crypto/pem/PemKeyStoreTest.java

```diff
@@ -10,24 +10,25 @@
  *******************************************************************************/
 package de.dentrassi.crypto.pem;
 
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
-
 import java.security.KeyStore;
 import java.security.KeyStoreException;
 import java.security.Security;
 
+import org.assertj.core.api.Assertions;
 import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
+@Disabled
 public class PemKeyStoreTest {
 
 	@BeforeAll
 	public static void setup() {
 	}
 
 	protected void testWithProvider(final ThrowingCallable callable) throws Throwable {
-		assertThatThrownBy(() -> KeyStore.getInstance("PEM")).isInstanceOf(KeyStoreException.class);
+		Assertions.assertThatThrownBy(() -> KeyStore.getInstance("PEM")).isInstanceOf(KeyStoreException.class);
```

Comment:
Could you please leave the static method imports. Same for the second one in this file.